src/device-api/v1.js

```diff
@@ -0,0 +1,126 @@
+import * as Promise from 'bluebird';
+import * as _ from 'lodash';
+import * as constants from '../lib/constants';
+import { checkInt, checkTruthy } from '../lib/validation';
+import { doRestart, doPurge, serviceAction } from './common';
+
+export const createV1Api = function(router, applications) {
+	const { eventTracker } = applications;
+
+	router.post('/v1/restart', function(req, res, next) {
+		const appId = checkInt(req.body.appId);
+		const force = checkTruthy(req.body.force) ?? false;
```

Comment:
I had to add the `?? false` here to pass the type checks, previously `force` could be `undefined` - is `false` the right default/

Reply:
It is yeah.